### PR TITLE
Add makefile, enforce lint, fix lint errors.

### PR DESCRIPTION
**Pull request by bogdandrutu**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -54,3 +55,9 @@ $ git push fork feature
 ```
 
 Open a pull request against the main opencensus-go repo.
+
+## General Notes
+This project uses Go 1.11.4 and Travis for CI.	This project uses Appveyor and Travis for CI.
```
Comment (songy23) [inline CONTRIBUTING.md]: nit: remove the first "and Travis for CI".

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -54,3 +55,9 @@ $ git push fork feature
...
+
+The dependencies are managed with `go mod` if you work with the sources under your
+`$GOPATH` you need to set the environment variable `GO111MODULE=on`.	`$GOPATH` you need to set the environment variable `GO111MODULE=on`.
```
Comment (songy23) [inline CONTRIBUTING.md]: Same here, duplicates.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bogdandrutu) [reply CONTRIBUTING.md]: Done.

File: CONTRIBUTING.md (hunk 2 above)
Comment (bogdandrutu) [reply CONTRIBUTING.md]: Done.

**Comment by rghetia** [comment]
LGTM

**Pull request merged**